### ISSUE TITLE
Move /connect to queue instead of waiting for success database update

### ISSUE THREAD

**Issue by xmflsct** [tooot-app/push#4]

**Comment by xmflsct** [comment]
This does not work as we need to inform users of not found token, also redis cache cannot be used in this case. Need to have this logic in the app that checks token only once per day when people open the app many times per day.

**Comment by xmflsct** [comment]
Won't fix for now
